Bottom: eda57ff11e0a7aa5982a6d9d0eddd8581807288e
Top:    278a7293c895b8e53d2127129bc43cbfe08b33ed
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 02:45:13 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 946668f..d2831a3 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1175,6 +1175,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1193,6 +1194,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
